rules/framework.bzl

```diff
@@ -512,8 +512,28 @@ def _merge_root_infoplists(ctx):
 
     return output_plist
 
+def _attrs_for_split_slice(attrs_by_split_slices, split_slice_key):
+    if len(attrs_by_split_slices.keys()) == 0:
+        return []
+    elif len(attrs_by_split_slices.keys()) == 1:
+        return attrs_by_split_slices.values()[0]
+    else:
+        return attrs_by_split_slices[split_slice_key]
```

Comment:
Should this fail `if split_slice_key not in attrs_by_split_slices`?

Reply:
The [] operator will fail if the key is not in the dict.